README.md

```diff
@@ -40,6 +40,40 @@ This tap:
 
  - Only designed to work with Xero [Partner Applications](https://developer.xero.com/documentation/auth-and-limits/partner-applications), not Private Applications.
 
+## Getting started
+
+### Configuration
+
+The configuration should take the following form:
+
+```json
+{
+  "start_date": "2020-04-01",
+  "client_id": "...",
+  "client_secret": "...",
+  "tenant_id": "...",
+  "refresh_token": "..."
+}
+```
+
+Choose your preferred start date. You will need to get the other 4 values from the Xero API.
+
+ * Client ID and Secret are provided by the [Xero developer portal](http://developer.xero.com),
+   if you create a new OAuth2 app.
+ * To get the tenant ID and refresh token, you will need to set up a web app
+   to receive the OAuth2 repsonse. A simple script for this is available at
+   [sminnee/tap-xero-config](http://github.com/sminnee/tap-xero-config).
+
+### Running without installation
+
+When making improvements to tap-xero, you will likely to want to run the 
+script without installing the pacakge. You can do this with the following
```

Comment:
Are you doing to this because you want to be able to run your edited code?

Reply:
Sorry for the late reply - I missed the notification!

Yes, I was making changes to tap-xero, and required a functioning local dev environment connected to my own Xero account. I found this to be a little non-trivial and ended up adapating some PHP code for doing and OAuth handshake with the xero servers to dump the necessary keys.

I wrote it up in this PR to assist anyone else who might want to contribute to tap-xero.

Reply:
Yes, this command lets people run a checked-out copy of tap-xero with their local modifications. It may be obvious to experienced Python developers but it wasn't to me. 😉

Reply:
I'm glad you found your way back to this PR even though you missed my comment notification!

I should've just left this in my first comment: I definitely agree with you and see the need to run the changed code. While there's nothing wrong with the way you did it - obviously since you were able to iterate on the code with it - there is a better way.

So `pip` lets you install local files instead of reaching out to like PyPI or Github to grab a package. This is [editable mode](https://pip.pypa.io/en/stable/cli/pip_install/#install-editable).

It's really just a wrapper on `setuptools`'s [Develop mode](https://setuptools.readthedocs.io/en/latest/userguide/development_mode.html). 

The benefit I see of doing it this way is that if you have dependencies in the script you are trying to run, you are more likely to have them installed because they were listed in the project's setup file. I'm not sure how you got around the `singer` dependency in `tap_xero.__init__`; you must have it installed somewhere.

Using development mode also makes your run command less cryptic. It could be as simple as 
```sh
tap-xero -c config.json -p catalog.json
```